addons/mail/static/src/new/messaging_menu/messaging_menu.xml

```diff
@@ -21,9 +21,9 @@
         'o-mail-messaging-menu-topbar border-bottom d-flex flex-shrink-0 p-2': !env.inDiscussApp,
         }">
             <t t-if="!store.isSmall">
-                <button class="btn btn-link" t-att-class="store.discuss.activeTab === 'all' ? 'fw-bolder' : 'text-500'" type="button" role="tab" t-on-click="() => store.discuss.activeTab = 'all'">All</button>
-                <button class="btn btn-link" t-att-class="store.discuss.activeTab === 'chat' ? 'fw-bolder' : 'text-500'" type="button" role="tab" t-on-click="() => store.discuss.activeTab = 'chat'">Chats</button>
-                <button class="btn btn-link" t-att-class="store.discuss.activeTab === 'channel' ? 'fw-bolder' : 'text-500'" type="button" role="tab" t-on-click="() => store.discuss.activeTab = 'channel'">Channels</button>
+                <button class="o-mail-messaging-menu-tab btn btn-link" t-att-class="store.discuss.activeTab === 'all' ? 'fw-bolder o-active' : 'text-500'" type="button" role="tab" data-tab-id="all" t-on-click="() => store.discuss.activeTab = 'all'">All</button>
```

Comment:
General comment for PR: instead of adding `data-` and `class` again what we should do in tests is to adapt selectors to use more realistic information (text content with `contains`, stuff like that).